src/gbcc_video.c

```diff
@@ -9,13 +9,16 @@
 #define BACKGROUND_MAP_BANK_2 0x9C00u
 #define NUM_SPRITES 40
 
+enum palette_flag { bk, sp1, sp2 };
```

Comment:
Enum values should be all caps, as done elsewhere, and more descriptive e.g. `{ BACKGROUND, SPRITE_1, SPRITE_2 }`